examples/display_summed_images.py

```diff
@@ -37,58 +37,65 @@ class ImageSumDisplayerTool(Tool):
                     'telgroup': 'ImageSumDisplayerTool.telgroup',
                     'max-events': 'ImageSumDisplayerTool.max_events',
                     'output-suffix': 'ImageSumDisplayerTool.output_suffix'})
-    classes = List([CameraCalibrator,])
+    classes = List([CameraCalibrator,HessioFileReader])
 
     def setup(self):
         # load up the telescope types table (need to first open a file, a bit of
         # a hack until a proper insturment module exists) and select only the
         # telescopes with the same camera type
-        data = next(hessio_event_source(self.infile, max_events=1))
-        data.inst.subarray.info(printer=self.log.info)
-        camtypes = data.inst.subarray.to_table().group_by('camera_type')
+
+        self.reader = HessioFileReader(None, None, input_url=self.infile,
+                                       max_events=self.max_events)
```

Comment:
Indenting like this:
```python
self.reader = HessioFileReader(
    None, 
    None, 
    input_url=self.infile,
    max_events=self.max_events,
)
```
instead of this:
```python
self.reader = HessioFileReader(None, None, input_url=self.infile,
                               max_events=self.max_events)
```

Has some advantages:
 * stability over refactoring, when renaming `HessioFileReader` to `HessioFile` or `HessioEventSource` in the future, the 2nd line does not need to be re-indented.
 * the max-line length in pep8 is 80 chars, but most people like to read between 50...60 chars, hence the width of most magazine or newspaper columns. 
 * the latter is *discouraging* the use of explanatory keyword arguments, while the former can easily transition into:
self.reader = HessioFileReader(
    config=None, 
    tool=None, 
    input_url=self.infile,
    max_events=self.max_events,
)

Which is even better readable.

 * The latter way of indenting optically  "groups" the first 3 parameters, but they are no group.